src/endpoints/deleteUser.ts

```diff
@@ -0,0 +1,53 @@
+import { Request, Response } from "express";
```

Comment:
Please add a next function

Reply:
This was now resolved in [7717d15](https://github.com/Chillandchat/api/pull/8/commits/7717d15a97872b27d6b59822cfbe1af0ccffaea3)